Sources/PrimerSDK/Classes/Core/Analytics/Device.swift

```diff
@@ -31,7 +32,14 @@ struct Device: Codable {
         batteryStatus = batteryLevel == -100 ? "CHARGING" : "NOT_CHARGING"
         UIDevice.current.isBatteryMonitoringEnabled = false
         
-        
+        if let languageCode = Locale.current.languageCode {
+            if let regionCode = Locale.current.regionCode {
+                self.locale = "\(languageCode)-\(regionCode)"
+            } else {
+                self.locale = "\(languageCode)"
+            }
+        }
+        self.locale = Locale.current.languageCode
```

Comment:
Doesn't this mean locale is always set to this value, overwriting the logic above?